openshift/log-detective.yaml

```diff
@@ -31,7 +32,7 @@ spec:
               cpu: "50m"
             limits:
               memory: "800Mi"
-              cpu: "1"
+              cpu: "0.5"
```

Comment:
The quota we have is "1" and I don't think we need that much for this pod. Anyway, I had to lower this one so that I could run the temp pod to copy the TLS certs in the PV.

I will revert this change because in the end it's not actually needed.